lib/lib.h

```diff
@@ -9,6 +10,12 @@ void get_sha_256(unsigned char out[crypto_hash_sha256_BYTES]) {
   int fd;
   int n_read;
 
+  // Initialize sodium.h
```

Comment:
I may need to initialize sodium in some initialize function -- however there's no harm (https://download.libsodium.org/doc/usage/) in calling `sodium_init()` multiple times since no additional file descriptors are created.